tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -604,10 +609,10 @@ describe('runs_table', () => {
 
           expect(
             items.map((element) => element.getAttribute('aria-checked'))
-          ).toEqual(['true', 'false', 'false']);
+          ).toEqual(['true', 'false', 'false', null]);
```

Comment:
Shouldn't the last one be `aria-checked: false` and not `null`? If possible, I think we should check a11y spec to see what type of roles we are supposed to use for this type of UI.

Reply:
add `aria-checked` in html